docs/pages/setup/reference/cli.mdx

```diff
@@ -1305,6 +1305,360 @@ $ tctl auth rotate --type=user --grace-period=200h
 ```
 
+### tctl sso test
+
+Perform end-to-end test of SSO flow using provided auth connector definition.
+
+The command supports all auth connector types: `github`, `oidc`, `saml`. The latter two require Teleport Enterprise.
+
+The testing consists of running a single end-to-end authentication request against provided auth connector definition.
+Once the request is finished, the results will be printed to console along with context-specific diagnostic informations.
+The test process will modify the list of configured auth connectors or result in certificates being issued.
+
+```code
+$ tctl [<global-flags>] sso test [<filename>]
+```
+
+<Admonition
+  type="note"
+  title="Important"
+>
+  To use this command, you must have access to `github_request`, `oidc_request`, `saml_request` resources (depending on the type of connector being tested).
+
+  If you receive "permission denied" error, modify one of your roles to allow access to:
+
+```yaml
+   - resources:
+      - github_request
+      verbs:
+      - list
+      - create
+      - read
+      - update
+      - delete
+    - resources:
+      - oidc_request
+      verbs:
+      - list
+      - create
+      - read
+      - update
+      - delete
+    - resources:
+      - saml_request
+      verbs:
+      - list
+      - create
+      - read
+      - update
+      - delete
+```
+</Admonition>
+
+
+#### Arguments
+
+- `[<filename>]` Connector resource definition file. Optional. Empty for stdin.
+
+#### Flags
+
+This command defines no flags.
+
+#### Global flags
+
+These flags are available for all commands `--debug, --config`. Run
+`tctl help <subcommand>` or see the [Global Flags section](#tctl-global-flags).
+
+#### Examples
+
+Test the auth connector from `connector.yaml`:
+
+```code
+$ tctl sso test connector.yaml
+```
+
+The command is designed to be used in conjunction with `tctl sso configure` family of commands:
+
+```code
+$ tctl sso configure ... | tctl sso test
+```
+
+The pipeline may also utilise `tee` to capture the connector generated with `tctl sso configure`. This will save the connector in `connector.yaml`:
+
+```code
+$ tctl sso configure ... | tee connector.yaml | tctl sso test
+```
+
+You can test existing auth connector by combining the command with `tctl get`:
+
+```bsh
+$ tctl get saml/your-connector-name --with-secrets | tctl sso test
+````
+
+<Admonition
+  type="note"
+  title="Important"
+>
+  Make sure to include `--with-secrets` flag, or the exported auth connector will not be testeable.
+</Admonition>
+
+### tctl sso configure github
+
+Configure Github auth connector.
+
+Required params `--id` and `--secret` come from the [Github OAuth app](https://docs.github.com/en/developers/apps/building-oauth-apps/creating-an-oauth-app).
+
+The flag `--teams-to-logins` can be provided multiple times to specify which Github Teams are assigned which roles.
+
+```code
+$ tctl sso test --id=GITHUB_CLIENT_ID --secret=GITHUB_SECRET --teams-to-logins=... [<other-flags>]
```

Comment:
Can you give an example of `--teams-to-logins` rather than `...` which makes the user have to guess?

Is it something like `--teams-to-logins foo:bar`?